Bottom: 8d67e4dcbaabc459b38b3cc8c22b693338b51d0a
Top:    b133f392278208f13d214f109850a01cb0c249fc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-07 09:56:04 +0000

Refresh of add-support-for-a-poured-objec

---

diff --git a/src/change.c b/src/change.c
index f0d7e69..0b5f129 100644
--- a/src/change.c
+++ b/src/change.c
@@ -611,7 +611,6 @@ ChangePadClearSize (ElementTypePtr Element, PadTypePtr Pad)
 
   if (TEST_FLAG (LOCKFLAG, Pad))
     return (NULL);
-<<<<<<< current:src/change.c
   value = MIN (MAX_LINESIZE, value);
   if (value < 0)
     value = 0;
